sources/mysql/mysqldump_test.go

```diff
@@ -819,12 +819,12 @@ func TestProcessMySQLDump_DataError(t *testing.T) {
 			expectedData: []spannerData{
 				spannerData{
 					table: "test", cols: []string{"a", "b", "c", "d", "e", "f", "g", "synth_id"},
-					vals: []interface{}{int64(7), float64(42.1), true,
+					vals: []interface{}{int64(7), float32(42.1), true,
```

Comment:
Please add a double column and set a float64 value too

Reply:
done.